core/templates/services/site-analytics.service.ts

```diff
@@ -46,16 +46,6 @@ export class SiteAnalyticsService {
 
   // For definitions of the various arguments, please see:
```

Comment:
I think you can delete this comment. The link no longer works.

Reply:
Sure, I will do that.

@seanlip How can we confirm this https://github.com/oppia/oppia/pull/20661#pullrequestreview-2176226617

Reply:
We do have some dashboards based on the old events. However, old GA is deprecated fully so those dashboards are broken anyway. So removing the old GA should not affect anything.

/cc @nikitaevg

Reply:
@seanlip Can we proceed to merge this PR as soon as all CI checks are done?

Reply:
Yes, should be OK.